app/models/concerns/annotation_base.rb

```diff
@@ -24,8 +24,6 @@ def define_annotation_relation_method
           query = self.annotation_query(type, context)
           klass = (type.blank? || type.is_a?(Array)) ? Annotation : type.camelize.constantize
           relation = klass.where(query)
-          relation = relation.where(context_id: nil) if context == 'none'
-          relation = relation.where.not(context_id: nil) if context == 'some'
           relation.order('id DESC')
```

Comment:
The idea of those conditions was to get the annotations given a project, which is not a context anymore, but an annotated, right? So, should we really remove those two lines or should we replace `context_id` by `annotated_id`?